### PR TITLE
Update Istio version to 1.4.6

### PR DESCRIPTION
**Pull request by nak3**
Part of https://github.com/knative/serving/issues/7568

## Proposed Changes

This patch updates Istio version to 1.4.6 which is tested CI and
stable version.

/cc @tcnghia

### CODE REVIEW

File: docs/install/installing-istio.md [hunk 1]
```diff
@@ -13,9 +13,6 @@ installation. If your cloud platform offers a managed Istio installation, the
 [install guide](./README.md) for your specific platform will have those
 instructions.
 
-For example, the [GKE Install Guide](./Knative-with-GKE.md) includes the
```
Comment (nak3) [inline docs/install/installing-istio.md:16]: I have removed this as the specific installation like GKE https://github.com/knative/docs/commit/dcaf0b801cda65f1d8ecec736cf479b60eaa7ebb is already removed.

**Comment by tcnghia** [comment]
@mattmoor can you please /approve

File: docs/install/installing-istio.md [hunk 2]
```diff
@@ -342,5 +339,5 @@ rm -rf istio-${ISTIO_VERSION}
 [2]:
   https://istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#manual-sidecar-injection
 [3]: https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/
-[4]: https://istio.io/docs/tasks/security/mutual-tls/
+[4]: https://istio.io/pt-br/docs/concepts/security/#mutual-tls-authentication
```
Comment (evankanderson) [inline docs/install/installing-istio.md:342]: I don't think you need the `/pt-br/` language-specific docs (which doesn't seem to work anyway).

Comment (nak3) [reply docs/install/installing-istio.md]: Sorry my bad. Removed `pt-br`.

**Pull request merged**